app/helpers/evaluations_helper.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+# View helpers for calculating evaluation & submission details.
+module EvaluationsHelper
+  STATUS_COLORS = {
+    not_started: 'bg-error-dark',
+    in_progress: 'bg-accent-warm-dark',
+    completed: 'bg-success-dark',
+    recused: 'bg-base',
+    unassigned: 'bg-base',
+    recused_unassigned: 'bg-base'
+  }.freeze
+
+  Score = Struct.new(:raw_score, :formatted_score, :display_score)
+
+  def evaluation_submission_assignment_status_color(assignment)
+    STATUS_COLORS[assignment.evaluation_status]
+  end
+
+  def display_score(assignment)
+    return 'N/A' unless assignment.evaluation_status == :completed
+
+    assignment.evaluation&.total_score || 'N/A'
+  end
+
+  # individual evaluator score
+  def evaluator_score(assignment)
+    score = assignment.evaluation&.total_score
+
+    unless assignment.evaluation_status == :completed && score
+      return Score.new(0, "0", "N/A")
+    end
+
+    Score.new(score, score.to_s, score)
+  end
+
+  def average_score(submission)
+    completed_evaluations = submission.evaluations.where.not(completed_at: nil)
+
+    unless completed_evaluations.any?
+      return Score.new(0, "0", "N/A")
+    end
+
+    avg = completed_evaluations.average(:total_score)
+    score = avg ? avg.round : 0
+    Score.new(score, "#{score}%", "#{score}%")
```

Comment:
the average score should not have a percent sign after it. the score will not be a percentage for non-weighted-scale scores. figma is inconsistent here but i'm sure we don't want to show a score like "318%". requested clarification on figma.

Reply:
just confirmed, we can remove the % signs from the scores 👍